stl/src/xmtx.cpp

```diff
@@ -11,24 +11,24 @@ _EXTERN_C_UNLESS_PURE
 
 // Win32 critical sections are recursive
 
-void __CLRCALL_PURE_OR_CDECL _Mtxinit(_Rmtx* _Mtx) noexcept { // initialize mutex
+_MRTIMP2_PURE void __CLRCALL_PURE_OR_CDECL _Mtxinit(_Rmtx* _Mtx) noexcept { // initialize mutex

```

Comment:
This is good, but we should simplify `xmtx.hpp` accordingly.

Reply:
That would be merge conflict with #3900

Reply:
Trivial to resolve, thanks for the heads up.